lunes_cms/cms/models/static.py

```diff
@@ -11,13 +11,17 @@ class Static:
     """
 
     # Possible articles
-    article_choices = [
+    singular_article_choices = [
         (0, "keiner"),
         (1, "der"),
         (2, "die"),
         (3, "das"),
         (4, "die (Plural)"),
```

Comment:
I think it makes to open an issue for in a few months to refactor this and remove (4, "die (Plural)")

Reply:
Isn't `die (Plural)` needed for words that appears only in plural (like "Ferien")?

Reply:
As memo for us in future #519 is opened